src/main/java/seedu/address/logic/Logic.java

```diff
@@ -38,6 +38,10 @@ public interface Logic {
 
     /** Return the desired filtered item */
     Item getFilteredItem();
+
+    /** Returns an unmodifiable view of the filtered list of items */
```

Comment:
seems like this is getting the actual list itself, so it's not really unmodifiable